LayoutTests/http/tests/site-isolation/mouse-events/iframes-with-tooltips.html

```diff
@@ -0,0 +1,81 @@
+<!-- webkit-test-runner [ SiteIsolationEnabled=true ] -->
+<script src="/js-test-resources/js-test.js"></script>
+<script src="/js-test-resources/ui-helper.js"></script>
+<script>
+description("Verifies that tooltips from iframes are correct.");
+jsTestIsAsync = true;
+
+async function runTest() {
+
+    if (window.testRunner) {
+        {
+            let promise = testRunner.tooltipDidChange();
+            await eventSender.asyncMouseMoveTo(30, 30);
+            result = await promise;
+            result = await promise;
+            shouldBe("result", "'Test Main Frame'");
+        }
+        {
+            let promiseClearing = testRunner.tooltipDidChange();
+            eventSender.asyncMouseMoveTo(100, 100);
```

Comment:
not awaiting these async mouse moves is strange, is that intentional?

Reply:
nope, mistake, thanks!